sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib 5447906b1c3be5b6189e61c6256dc75c291c59e5
+github.tarball_from archive
+
+name                portfile-gen
+version             0.2
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           any
+license             BSD
+supported_archs     noarch
+
+description         Generate a basic template Portfile given a few bits \
+                    of information
+long_description    {*}${description}.
+homepage            https://github.com/${github.author}/${github.project}/blob/master/${name}
+
+livecheck.type      none
+
+fetch.type          git
+
+worksrcdir          ${name}
+
+use_configure       no
+
+# should be implied by turning off configure:
+configure.ccache    no
+
+build {
+    ui_debug "${name} has no build step"
+}
```

Comment:
same as in the other PR - remove useless message

Reply:
It's not useless, it's helpful and informative. All ports that override build steps should explicitly tell me when they're doing so, IMO.

Reply:
Added an extra comment in 7907811 telling people not to remove it.

Reply:
Please remove as requested

Reply:
It's necessary to prevent that block from being empty, which is something that's warned about in other programming languages, see for example the gcc/clang warning `-Wempty-body` for the C family of languages, or luacheck's W542 for lua. It's also necessary to avoid violating the principle of least surprise; a user may wonder why a step is getting overridden, and this makes it clear what's going on by explaining it.

Reply:
This is not done in other Portfiles, and there is no need for it. The fact that some programming languages warn about empty blocks isn't really germane.

Reply:
Well it *ought* to be done in other Portfiles. There is a desire for it.

Reply:
Maintaining MacPorts to a high degree of quality requires a lot of work on the part of people like @reneeotten who merge huge numbers of pull requests. It is necessary to have uniformity and reasonably standard practice in order to make review fast and efficient. Something that only adds a couple of minutes to a review adds hours to his monthly burden. So it does matter.

Reply:
> Maintaining MacPorts to a high degree of quality requires a lot of work on the part of people like @reneeotten who merge huge numbers of pull requests. It is necessary to have uniformity and reasonably standard practice in order to make review fast and efficient. Something that only adds a couple of minutes to a review adds hours to his monthly burden. So it does matter.

Just say yes, it's not that hard.

Reply:
> It's easy to make work for other people, it's a lot harder to do that work.

Clicking the "approve" button takes no more than a second, less time than I spent writing the line in question. Also I can help review PRs if that's what you need.

Reply:
You can't just click approve, you have to actually go through and make sure that the thing _can_ be approved because it's correct. The simpler, more stereotypical, and easier to understand a request is, the less time it takes to figure out if you can approve it or not. This is why we need Portfiles to be simple, comprehensible, and to follow stereotyped patterns.

Reply:
It *is* correct, though. I don't see what's difficult to understand about a single `ui_debug` statement; that's the stereotypical pattern for providing extra information that only some users might want to see.